opentech/apply/activity/models.py

```diff
@@ -9,6 +9,14 @@
     ACTION: 'Action',
 }
 
+ALL = 'all'
+PRIVATE = 'private'
+
+VISIBILITY = {
+    ALL: 'All',
+    PRIVATE: 'Internal',
+}
+
```

Comment:
As discussed:

`PUBLIC: 'Public'`, `PRIVATE: 'Private'` and then `INTERNAL: 'Internal'` (for OTF-only comms in a followup feature)

Reply:
`PUBLIC: 'Public', REVIEWERS: 'Reviewers' and then INTERNAL: 'Internal'`